### PR TITLE
feat(deployer): add the csi-node plugin to the deployer

### PR DESCRIPTION
**Pull request by tiagolobocastro**
feat(deployer): add the csi-node plugin to the deployer

The csi-node originally came from the data-plane repo and so was never part of the deployer.

---

test(bdd): deploy the csi-node using the deployer

---

refactor: name silence log env var more appropriately

Rename quiet to silence, which seems more appropriate.

### CODE REVIEW

**Review by Abhinandan-Purkait — APPROVED** [review]

File: deployer/src/infra/csi_node.rs [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+            CsiNode::wait_app_node(i).await?;
+        }
+
+        Ok(())
+    }
+}
+
+impl CsiNode {
+    fn name(i: u32) -> String {
+        format!("app-node-{}", i + 1)
```
Comment (paulyoong) [inline deployer/src/infra/csi_node.rs:87]: Should this be "csi-node-{}"?

**Review by paulyoong — APPROVED** [review]

File: deployer/src/infra/csi_node.rs (hunk 1 above)
Comment (tiagolobocastro) [reply deployer/src/infra/csi_node.rs:87]: I left it as app-node-{} as this is "simulating" an application node (which is where the csi-node runs)

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors cancel

**Comment by tiagolobocastro** [comment]
bors merge

**Pull request merged**